AI_GLOSSARY.md

```
@@ -0,0 +1,1170 @@
+9. [Built-in Bindings — Control Flow](#built-in-bindings--control-flow)
+10. [Built-in Bindings — Components & Templates](#built-in-bindings--components--templates)
+11. [Virtual Elements (Comment Bindings)](#virtual-elements-comment-bindings)
+12. [Provider System](#provider-system)
+13. [Component System](#component-system)
+14. [JSX / TSX Path](#jsx--tsx-path)
+15. [LifeCycle](#lifecycle)
+16. [DOM Utilities](#dom-utilities)
+17. [Tasks / Microtask Scheduler](#tasks--microtask-scheduler)
+18. [MVVM Pattern Terms](#mvvm-pattern-terms)
+19. [Build & Architecture Terms](#build--architecture-terms)
+20. [Governance & Process Terms](#governance--process-terms)
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)